src/Backend/Modules/Location/Domain/Location.php

```diff
@@ -0,0 +1,295 @@
+<?php
+
+namespace Backend\Modules\Location\Domain;
+
+use Common\Locale;
+use Backend\Core\Language\Locale as BackendLocale;
+use Frontend\Core\Language\Locale as FrontendLocale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="location")
+ * @ORM\Entity
+ * @ORM\HasLifecycleCallbacks
+ */
+class Location
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @var Locale
+     *
+     * @ORM\Column(type="locale", name="language")
+     */
+    private $locale;
+
+    /**
+     * @TODO: map this to some sort of Extra entity
+     * @var integer
+     *
+     * @ORM\Column(type="integer", name="extra_id")
+     */
+    private $extraId;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $title;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $street;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $number;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $zip;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $city;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string")
+     */
+    private $country;
+
+    /**
+     * @var float
+     *
+     * @ORM\Column(type="float", name="lat")
+     */
+    private $latitude;
+
+    /**
+     * @var float
+     *
+     * @ORM\Column(type="float", name="lng")
+     */
+    private $longitude;
+
+    /**
+     * @var bool
+     *
+     * @ORM\Column(type="boolean", options={"default" = true}, name="show_overview")
+     */
+    private $showInOverview;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(type="datetime", name="created_on")
+     */
+    private $createdOn;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(type="datetime", name="edited_on")
+     */
+    private $editedOn;
+
+    public function __construct(
+        Locale $locale,
+        string $title,
+        string $street,
+        string $number,
+        string $zip,
+        string $city,
+        string $country,
+        float $latitude,
+        float $longitude,
+        bool $showInOverview = true,
```

Comment:
Niet logischer om die niet in constructor te steken? Maar met zo static-methods te werken:
`public function createHiddenInOverview()`?

Reply:
I don't completely follow with what you're saying here

Reply:
As discussed, not that relevant.